dev/helm/templates/deployment.yaml

```diff
@@ -53,6 +53,7 @@ spec:
               value: "{{ default "false" .Values.postgresql.ssl }}"
             - name: DB_SSL_CA
               value: "{{ default "" .Values.postgresql.ca }}"
+            {{- if .Values.postgresql.enabled -}}
```

Comment:
If DATABASE_URL is defined, all other DB_ vars will be ignored so there's no need for this condition.

Reply:
SSL?
I might be in the wrong place: (not extracted from ?sslmode=require in DATABASE_URL)

https://github.com/Kencove/wiki/blob/main/server/core/db.js#L32

But seems to be tested again. 
https://github.com/Kencove/wiki/blob/main/server/core/db.js#L42

Reply:
Since dbConfig is a string (from DATABASE_URL), all SSL code afterwards is not actually used for the db connection.

e.g. options are ignored at https://github.com/Kencove/wiki/blob/main/server/core/db.js#L82 because it's not an object.

The string from DATABASE_URL is passed directly at https://github.com/Kencove/wiki/blob/main/server/core/db.js#L133

so any additional params like `?sslmode=require` would be parsed by the DB driver at this point.